cli/internal/runsummary/run_summary.go

```diff
@@ -16,7 +25,21 @@ type RunSummary struct {
 	TurboVersion      string             `json:"turboVersion"`
 	GlobalHashSummary *GlobalHashSummary `json:"globalHashSummary"`
 	Packages          []string           `json:"packages"`
-	Tasks             []TaskSummary      `json:"tasks"`
+	Tasks             *[]*TaskSummary    `json:"tasks"`
+}
+
+// Save saves the run summary to a file
+func (summary *RunSummary) Save(singlePackage bool) error {
+	json, err := summary.FormatJSON(singlePackage)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("[debug] got my json for run summar\n")
+
+	nonce := "abcdef"
+	filename := fmt.Sprintf("turbo-run-%s-%s.json", nonce, time.Now().Format(time.RFC3339))
```

Comment:
This needs to be prefixed by `vercel-` when in the Vercel CI env (`VERCEL=1`) for us to pick it up automatically.

Reply:
done